pyproject.toml

```
@@ -30,7 +30,7 @@ classifiers = [
 dependencies = [
     "anyio >=3.6.2,<5",
     "sqlite-anyio >=0.2.3,<0.3.0",
-    "pycrdt >=0.9.16,<0.10.0",
+    "pycrdt >=0.10.0,<0.11.0",
```

Let's take the latest bugfix.
I'm also wondering if we should wait for the implementation of periodic awareness updates in pycrdt?
```suggestion
    "pycrdt >=0.10.1,<0.11.0",
```

---

I don't think it is required for this change.
But we should probably remove the `type != "update"` condition to broadcast all the event.

---

Right, sorry again for the confusion. The `change` event is only for observers, but not for clients...

---

OK, I'll merge then.